### ISSUE TITLE
Evaluate on FuzzBench

### ISSUE THREAD

**Issue by wideglide** [ivicanikolicsg/SivoFuzzer#1]
Hi - just wondering if you have any interest in evaluating this fuzzer on Google FuzzBench?

**Comment by ivicanikolicsg** [comment]
Yes, we are in the (long) process of implementing it on FuzzBench
